lxd/events/common.go

```diff
@@ -126,17 +65,8 @@ func (e *listenerCommon) Close() {
 		return
 	}
 
-	logger.Debug("Event listener server handler stopped", logger.Ctx{"listener": e.ID(), "local": e.Conn.LocalAddr(), "remote": e.Conn.RemoteAddr()})
+	logger.Debug("Event listener server handler stopped", logger.Ctx{"listener": e.ID(), "local": e.EventListenerConnection.LocalAddr(), "remote": e.EventListenerConnection.RemoteAddr()})
 
-	e.Conn.Close()
+	e.EventListenerConnection.Close()
 	e.ctxCancel()
 }
-
-// WriteJSON message to the connection.
-func (e *listenerCommon) WriteJSON(v any) error {
```

Comment:
This already got removed in ` lxd/events/common: Remove unused WriteMessage function ` commit looks like another rebase issue.

Reply:
This is not a rebase issue. Those are different functions. `WriteMessage` was never ever used.

Reply:
Ahh OK my mistake.